test/unit/middlewares/application-validator.test.ts

```
@@ -133,6 +133,55 @@ describe("application validator test", function () {
+        },
+      };
+      const res = {
+        boom: {
+          badRequest: () => {},
+        },
+      };
+
+      const nextSpy = Sinon.spy();
+      await applicationValidator.validateApplicationUpdateData(req, res, nextSpy);
+      expect(nextSpy.callCount).to.equal(0);
+    });
```

ESLint

[error] Expected line break before this statement.

(mocha/consistent-spacing-between-blocks)